rest/config_test.go

```diff
@@ -264,8 +266,9 @@ func TestConfigValidationImportPartitions(t *testing.T) {
 			assert.NoError(tt, err)
 			errorMessages := config.setupAndValidateDatabases()
 			if test.err != "" {
-				require.Len(tt, errorMessages, 1)
-				assert.EqualError(tt, errorMessages[0], test.err)
+				require.NotNil(tt, errorMessages.ErrorOrNil())
```

Comment:
Is the NotNil check needed, or is the length check sufficient?

Reply:
Length check was not sufficient at this place because the subsequent call to errorMessages.Len() can be panic when setupAndValidateDatabases returns a nil reference in the absence of an error (i.e. in situations where validation is successful). As per the changes in the latest revision, we have modified the setupAndValidateDatabases signature to return an error and hence we need to type assert that error back to multierror.Error here to perform the subsequent assertions in the test.